bedrock/utils/management/cron_command.py

```diff
@@ -14,6 +14,8 @@
 from django.core.cache import cache
 from django.core.management.base import BaseCommand
 
+from bedrock.utils.management.decorators import alert_sentry_on_exception
```

Comment:
I'm not even sure we need this `CronCommand` class any longer. I think this is from a time before k8s and SQLite. Only the security advisories command is using it.

Reply:
Probably out of scope for this PR, but I don't think this class needs the decorator, only the `update_security_advisories.py` one does.

Reply:
Will ticket it for the grab bag